docs/api.js

```diff
@@ -41,44 +43,800 @@ export default {
+        parameters: [],
+        responses: {
+          '200': {
+            description: '',
+            headers: {},
+          },
+        },
+      },
+      post: {
+        description: 'Create a category.',
+        summary: 'Create category',
+        tags: ['Categories'],
+        operationId: 'CategoriesPost',
+        deprecated: false,
+        produces: ['application/json'],
+        consumes: ['application/x-www-form-urlencoded'],
+        parameters: [
+          {
+            name: 'Authorization',
+            in: 'header',
+            required: true,
+            type: 'string',
+            description: '',
+          },
+          {
+            name: 'name',
+            in: 'formData',
+            required: true,
+            type: 'string',
+            description: '',
+          },
+        ],
+        responses: {
+          '200': {
+            description: '',
+            headers: {},
+          },
+        },
+        security: [],
+      },
+    },
+    '/categories/frontend': {
+      get: {
+        description: 'Get a single category.',
+        summary: 'Get single category',
+        tags: ['Categories'],
+        operationId: 'CategoriesFrontendGet',
+        deprecated: false,
+        produces: ['application/json'],
+        parameters: [],
+        responses: {
+          '200': {
+            description: '',
+            headers: {},
+          },
+        },
+      },
+    },
+    '/technologies': {
+      get: {
+        description: 'Get all technologies.',
+        summary: 'Get all technologies',
+        tags: ['Technologies'],
+        operationId: 'TechnologiesGet',
+        deprecated: false,
+        produces: ['application/json'],
+        parameters: [],
+        responses: {
+          '200': {
+            description: '',
+            headers: {},
+          },
+        },
+      },
+      post: {
+        description: 'Add a technology.',
+        summary: 'Add technology',
+        tags: ['Technologies'],
+        operationId: 'TechnologiesPost',
+        deprecated: false,
+        produces: ['application/json'],
+        consumes: ['application/x-www-form-urlencoded'],
+        parameters: [
+          {
+            name: 'Authorization',
+            in: 'header',
+            required: true,
+            type: 'string',
+            description: '',
+          },
+          {
+            name: 'name',
+            in: 'formData',
+            required: true,
+            type: 'string',
+            description: '',
+          },
+          {
+            name: 'category',
+            in: 'formData',
+            required: true,
+            type: 'string',
+            description: '',
+          },
+        ],
+        responses: {
+          '200': {
+            description: '',
+            headers: {},
+          },
+        },
+        security: [],
+      },
+    },
+    '/technologies/javascript': {
+      get: {
+        description: 'Get a technology.',
```

Comment:
Could we have this description as `Get a single technology`?

Reply:
Of course